users/forms.py

```diff
@@ -67,7 +68,8 @@ class Meta:
 
 class CustomUserSignUpForm(UserCreationForm):
     email = forms.EmailField(max_length=constants.EMAIL_MAX_LENGTH, required=True, label="id_email")
+    captcha = CaptchaField()
 
     class Meta:
         model = CustomUser
-        fields = ("email", "password1", "password2")
+        fields = ("email", "password1", "password2", "captcha")
```

Comment:
If I'm not mistaken, this should contain all the fields that we want to import from the source model. So why is `captcha` mentioned here? It isn't a model field, is it? 😕

Reply:
Also non-model fields should be listed here.